### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_Keybaord: On activating the “Skip to content” link the focus gets updated to “license terms” link.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2797]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: https://code.visualstudio.com 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated : MAS 2.4.3, MAS 2.4.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate to “Skip to content” link and activate it
5.	Observe where it’s focus gets updated.

**Actual:**
When keyboard users navigate to the “Skip to content” link and activate it, the keyboard focus gets updated to the “license terms” link instead of moving to the first element in the main content.

The same issue is observed by the screen reader users also on activating the “Skip to content” link and activate it, the focus gets updated to the “license terms” link.

Similar issue was observed with Edge as well as in Chrome.

**Expected:**
The focus should be updated to the first element in the main content area on activating the “Skip to content” link.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
)
**User Impact:**
If the focus gets updated to any element in the main content area instead of the first element then the users will consider that element where the focus gets updated as the start of the main content area.

**MAS Reference:**
MAS 2.4.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}) 

MAS 2.4.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d})

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_Keybaord Focus order incorrect.pptx](https://github.com/microsoft/vscode-docs/files/3320381/Code.visualstudio.com_DownloadInsiders_Keybaord.Focus.order.incorrect.pptx)
